### ISSUE TITLE
"Edit on HackMD" button not displayed on Gitlab

### ISSUE THREAD

**Issue by jibe-b** [hackmdio/hackmd-it-issues#6]
I installed the latest version of the webextension on both Chromium (latest on Linux) and Firefox (latest on Linux)

and the "Edit on HackMD" button is not displayed on Gitlab as shown in

https://hackmd.io/s/hackmd-it-release-notes#v020-2018-06-17

but rather 
![image](https://user-images.githubusercontent.com/6146483/50982069-a5f32400-14fc-11e9-90fe-999d20e4dd1f.png)

**Comment by Trung-DV** [comment]
Btw, is that `hackmd-it` can working with private gitlab?

**Comment by themikesam** [comment]
not displayed on github too. 2019/7/4 13:50:10 GMT+8

**Comment by sheiun** [comment]
@DVTrung No, it doesn't. It only allows to access official GitHub and GitLab websites.
![圖片已上傳至：2019-9-22 23-14](https://user-images.githubusercontent.com/36428458/65390145-2017c000-dd8f-11e9-9c77-8fa889c75579.png)
